packages/omg-js-childchain/src/index.js

```diff
@@ -131,20 +130,20 @@ class ChildChain {
   }
 
   /**
-   * create, sign, build and submit transaction to the childchain using raw privatekey and single utxo
+   * create, sign, build and submit a transaction to the childchain using raw privatekey
    * 
    * @method sendTransaction
-   * @param {object} utxo - (only supports single utxo)
-   * @param {string} fromPrivateKey
-   * @param {string} toAddress
-   * @param {number} toAmount 
+   * @param {array} fromUtxos - array of utxos gathered from the watcher
+   * @param {string} fromPrivateKey - hex string private key
```

Comment:
`fromPrivateKeys` needs to be an array as well, because each utxo must be signed with a privateKey and these keys may be different.

Reply:
hmm, would that be pretty difficult to manage UX-wise ? I assume different utxos would also have to be signed with a correct corresponding privateKey